gym/spaces/__init__.py

```diff
@@ -1,3 +1,14 @@
+"""This module implements various spaces.
+
+Spaces describe mathematical sets and are used in Gym to specify valid actions and observations.
+Every Gym environment must have the attributes ``action_space`` and ``observation_space``.
+If, for instance, three possible actions (0,1,2) can be performed in your environment and observations
+are vectors in the two-dimensional unit cube, the environment code may contain the following two lines:
+
+::
+    self.action_space = spaces.Discrete(3)
```

Comment:
For code, I think you need to use `>>>` or `...` before the line

Reply:
Hm PyCharm does indeed not do syntax highlighting without those. However, I was under the impression that any indented text after :: will be considered code. I don't want to use `>>>` and `...` here because that bit of code doesn't make much sense in an interactive interpreter. Maybe we can wait to see how it plays with Sphinx and fix it in a second pass if necessary.